docs/docs/testing-your-assistant.mdx

```diff
@@ -3,22 +3,22 @@ id: testing-your-assistant
 sidebar_label: Testing Your Assistant
 title: Testing Your Assistant
 abstract: Rasa Open Source lets you test dialogues end-to-end by running through
-  test stories. You can also test the dialogue management and the message processing 
-  (NLU) separately.
+  test conversations. In addition to full test conversations, you can
```

Comment:
I believe we settled on `test stories` as the preferred term. See thread: https://rasa-hq.slack.com/archives/C016FLYV4QN/p1600259900015600

Reply:
Changed it to test stories.